src/patches/glog.patch

```diff
@@ -0,0 +1,47 @@
+diff --git a/fast_tokenizer/cmake/external/glog.cmake b/fast_tokenizer/cmake/external/glog.cmake
+index 2afc3960..fc2b21ce 100644
+--- a/fast_tokenizer/cmake/external/glog.cmake
++++ b/fast_tokenizer/cmake/external/glog.cmake
+@@ -21,17 +21,29 @@ SET(GLOG_REPOSITORY ${GIT_URL}/google/glog.git)
+ SET(GLOG_TAG        v0.4.0)
+ 
+ IF(WIN32)
+-  SET(GLOG_LIBRARIES "${GLOG_INSTALL_DIR}/lib/glog.lib" CACHE FILEPATH "glog library." FORCE)
++  SET(GLOG_LIBRARIES_RELEASE "${GLOG_INSTALL_DIR}/lib/glog.lib" CACHE FILEPATH "glog release library." FORCE)
++  SET(GLOG_LIBRARIES_DEBUG "${GLOG_INSTALL_DIR}/lib/glog${CMAKE_DEBUG_POSTFIX}.lib" CACHE FILEPATH "glog debug library." FORCE)
+   SET(GLOG_CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} /wd4267 /wd4530")
+   add_definitions("/DGOOGLE_GLOG_DLL_DECL=")
+ ELSE(WIN32)
+-  SET(GLOG_LIBRARIES "${GLOG_INSTALL_DIR}/lib/libglog.a" CACHE FILEPATH "glog library." FORCE)
++  SET(GLOG_LIBRARIES_RELEASE "${GLOG_INSTALL_DIR}/lib/libglog.a" CACHE FILEPATH "glog release library." FORCE)
++  SET(GLOG_LIBRARIES_DEBUG "${GLOG_LIBRARIES_RELEASE}" CACHE FILEPATH "glog debug library." FORCE)
```

Comment:
why debug postfix is not used for non-Windows?
As I see Apple also has non-empty debug postfix

Reply:
Enabled debug build for mac and linux too and changed the name